app/src/main/java/com/example/tasky/agenda/data/local/AgendaDatabase.kt

```diff
@@ -6,15 +6,17 @@ import com.example.tasky.agenda.data.local.model.AttendeeEntity
 import com.example.tasky.agenda.data.local.model.EventEntity
 import com.example.tasky.agenda.data.local.model.ReminderEntity
 import com.example.tasky.agenda.data.local.model.TaskEntity
+import com.example.tasky.agenda.data.local.model.relations.EventAttendeesCrossRef
 
 @Database(
     entities = [
         EventEntity::class,
         ReminderEntity::class,
         TaskEntity::class,
-        AttendeeEntity::class
+        AttendeeEntity::class,
+        EventAttendeesCrossRef::class
     ],
-    version = 1
+    version = 2
```

Comment:
This is just a personal thing I tend to do


Whenever I make updates to the DB, and the app wasn't released to the public (which is the case). I don't upgrade the DB version. I just uninstall the app and install it again

That way, the DB starts fresh, with the new entities


Again, just saves you from coding a whole migration, that will only affect you, and not your users

Reply:
Yes I'd do the same, migrations are only meant to be used when the app is already installed on real devices